public/doc/swagger-2-v0.1.0.yaml

```diff
@@ -2200,9 +2276,6 @@ definitions:
         readOnly: true
         title: Name
         type: string
-      resource_version:
```

Comment:
This shouldn't be removed

Reply:
Hm I assumed this was missing from the core models but it is there on ServicePlans, need to look at why the generator cleared this.

Reply:
Fixed

Reply:
Maybe you didn't have the latest version of `topological_inventory-core` when the generator was run?